docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -1,60 +1,28 @@
 weight: 600
 keywords:
   - metrics query
   - TraceQL metrics
 ---
 
-# TraceQL metrics queries
+# TraceQL metrics functions
 
-{{< docs/experimental product="TraceQL metrics" >}}
+<!-- If you add a new function to this page, make sure you also add it to the _index.md/functions section.-->
 
-TraceQL metrics is an experimental feature in Grafana Tempo that creates metrics from traces.
+TraceQL supports include `rate`, `count_over_time`, `quantile_over_time`, `histogram_over_time`, and `compare` functions.
 
-Metric queries extend trace queries by applying a function to trace query results.
-This powerful feature allows for adhoc aggregation of any existing TraceQL query by any dimension available in your traces, much in the same way that LogQL metric queries create metrics from logs.
+## Available functions
 
-Traces are a unique observability signal that contain causal relationships between the components in your system.
-Do you want to know how many database calls across all systems are downstream of your application?
-What services beneath a given endpoint are currently failing?
-What services beneath an endpoint are currently slow? TraceQL metrics can answer all these questions by parsing your traces in aggregate.
-
-![Metrics visualization in Grafana](/media/docs/tempo/metrics-explore-sample-2.4.png)
-
-## Enable and use TraceQL metrics
-
-You can use the TraceQL metrics in Grafana with any existing or new Tempo data source.
-This capability is available in Grafana Cloud and Grafana (10.4 and newer).
-
-To enable TraceQL metrics, refer to [Configure TraceQL metrics](https://grafana.com/docs/tempo/latest/operations/traceql-metrics/) for more information.
-
-## Exemplars
-
-Exemplars are a powerful feature of TraceQL metrics.
-They allow you to see an exact trace that contributed to a given metric value.
-This is particularly useful when you want to understand why a given metric is high or low.
-
-Exemplars are available in TraceQL metrics for all functions.
-To get exemplars, you need to configure it in the query-frontend with the parameter `query_frontend.metrics.exemplars`,
-or pass a query hint in your query.
-
-```
-{ name = "GET /:endpoint" } | quantile_over_time(duration, .99) by (span.http.target) with (exemplars=true)
-```
-
-## Functions
-
-TraceQL supports include `rate`, `count_over_time`, `quantile_over_time`, and `histogram_over_time` functions.
 These functions can be added as an operator at the end of any TraceQL query.
 
 `rate`
 : Calculates the number of matching spans per second
 
 `count_over_time`
-: Counts the number of matching spans per time interval (see the `step` API parameter)
+: Counts the number of matching spans per time interval (see the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics))
 
 `min_over_time`
 : Returns the minimum value of matching spans values per time interval (see the `step` API parameter)
```

Comment:
```suggestion
: Returns the minimum value for the specified attribute across all matching spans per time interval (see the `step` API parameter)
```

Reply:
The same is true for max_over_time. Its not the minimum value of spans, its the minimum value of some specific attribute (the attribute is provided as an input parameter by the user) on all spans passed as input to the function.

Reply:
Thank you for this information. I've updated the max and min over time with your updates in a separate edit.